kano_greeter/password_view.py

```diff
@@ -120,7 +122,7 @@ def _authentication_complete_cb(self, _greeter):
             logger.info('Login failed')
 
     def _auth_error_cb(self, text, message_type=None):
-        logger.info('There was an error logging in: {}'.format(text))
+        logger.info('There was an error logging in: {}'.format(encode(text)))
```

Comment:
The logger actually copes with unicode now. In fact, it is better to pass it a Unicode() object than to give it a utf-8 string, which is what we are doing here. So I'd suggest maing this format string a unicode.